FairSoft_test.cmake

```
@@ -65,24 +68,25 @@ if (USE_TEMPDIR)
     message(STATUS "${timestamp} ... done")
 else()
     # No temporary directory created
-    set(FS_TEST_WORKDIR "")
+    set(FS_TEST_WORKDIR ".")
```

I don't like that one, because `"."` is not correct in that case.

If `FS_TEST_WORKDIR` is empty/unset,  `build/test_workdir` is used.
```suggestion
    set(FS_TEST_WORKDIR "${CTEST_BINARY_DIRECTORY}/test_workdir")
```

---

I reverted to the old way (empty)